ui/js/service/http.service.ts

```diff
@@ -64,6 +64,15 @@ export class HTTPService {
     );
   }
 
+  authHeader(options?: any): string {
+    if (localStorage.getItem("jwt")) {
+        return "Bearer " + localStorage.getItem("jwt");
+    } else if (options.email && options.password) {
```

Comment:
You might need to change `options.email && options.password` to   `options?.email && options?.password` to resolve any potential undefined error when you the change the order